README.md

```diff
@@ -506,12 +506,38 @@ spec:
 The exact paths (keys) of parameter we need can be found by examining the event payload (eg: GitHub events).
 
 
-Run following command to apply Triggertemplate.
+Run following command to apply TriggerBinding.
 
 ```bash
 $ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/01_binding.yaml
 ```
 
+####  Trigger
+`Trigger` combines TriggerTemplate, TriggerBindings and interceptors. They are used as ref inside the EventListener.
+
+The definition of our Trigger is given in `03-triggers/04_trigger.yaml`.
+
+```yaml
+apiVersion: triggers.tekton.dev/v1alpha1
+kind: Trigger
+metadata:
+  name: vote-trigger
+spec:
+  serviceAccountName: pipeline
+  triggers:
+  - bindings:
+    - ref: vote-app
+    template:
+      name: vote-app
+```
+
+Run following command to apply Trigger.
+
+```bash
+$ oc create -f https://raw.githubusercontent.com/openshift/pipelines-tutorial/master/03_triggers/04_trigger.yaml
```

Comment:
These yaml files are numbered so that user knows in what order they should be applied (even though other order might work as well). However, README creates resources in this order: 2, 1, 4 and 3.

Reply:
OK.

Reply:
Changed.